tests/unit/EventCard.spec.js

```diff
@@ -49,5 +35,41 @@ describe( "EventCard", () => {
 
   it('should render the event card div', () => {
     expect(wrapper.classes()).toContain('event-card');
+  });
+
+  it('should render the heart button and heart image', () => {
+    const button = wrapper.find('.heart-btn')
+    const buttonImg = wrapper.find('.heart-image')
+
+    expect(button.exists()).toBe(true)
+    expect(buttonImg.exists()).toBe(true)
+  });
+
+  it('should render the event image', () => {
+    const image = wrapper.find('.main-image')
+
+    expect(image.exists()).toBe(true)
+  });
+
+  it('should render the event date', () => {
+    const date = wrapper.find('.event-date')
+
+    expect(date.exists()).toBe(true)
+  });
+
+  it('should render the event details button', () => {
+    const detailsBtn = wrapper.find('.event-details-btn')
+
+    expect(detailsBtn.exists()).toBe(true)
+  });
+
+  it('should change an evented to favorited when the heart button is clicked', () => {
+    const heartButton = wrapper.find('.heart-btn')
+
+    expect(wrapper.vm.favorited).toBe(false)
```

Comment:
Would love an explanation of this syntax at some point. I feel like I'm still getting the hang of this testing library and don't know all the syntax and tools yet.